tests/tls_e2e/README.md

```diff
@@ -0,0 +1,34 @@
+Unittests Transport Layer Security end-2-end scenarios
+=====================
+
+Test scenarios:
+
+In this test, we have three test hooks for enabling test scenarios.
+
+- fail_cert_verify_callback
+   When set to true, it will force a tls connecting party to fail cert_verify_callback unconditionally.
```

Comment:
For consistency, make tls as TLS in the document

Reply:
changed